lib/external_command/result.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+class ExternalCommand
+  class Result
+    attr_reader :output
```

Comment:
it's a bit of a red flag for one of the constructor parameters to be public, and for there to be no behaviour whatsoever in this class related to it.

As the only method is really only a very simple delegation to the _other_ constructor, I wonder if this really needs to be a full class at all, or whether a Struct might be enough for now. Or, perhaps this is a sign that this class should really be taking care of things like cleaning up the output, rather than returning it as-is for the caller to clean up?